include/nbl/asset/ICPUCommandBuffer.h

```diff
@@ -41,19 +41,42 @@ class ICPUCommandBuffer final :
         return nullptr; // TODO
     }
 
-    bool canBeRestoredFrom(const IAsset* _other) const
-    {
-        return false; // TODO
-    }
-
+    _NBL_STATIC_INLINE_CONSTEXPR auto AssetType = ET_COMMAND_BUFFER;
     E_TYPE getAssetType() const override
     {
-        return ET_COMMAND_BUFFER;
+        return AssetType;
```

Comment:
Even though this entire class is incomplete, ive changed the getAssetType to look similar to other IAsset derived classes

Reply:
we can probably nuke that entire class/header, we intend to have a GPU-Task-Graph instead to deal with synchronisation/layout transitions later